### DISCUSSION TITLE
Feel free to start a discussion here

### DISCUSSION THREAD

**Discussion by cajhin** [cajhin/capsicain#15] [General]
Never used this github feature. Might be better than Issues for discussing problems and possible features.

**Comment by MAnDumu** [comment]
Greetings, I don't know if this place is the correct one, I'm trying to do this but it does not work, guide does not especify it clearly


MACRO Q PLAYBACK --> SEQUENCE (4) {Av #2} {Zv #3} {A^ #2} {Z^ #1}

**Comment by MAnDumu** [comment]
I could copy the debug stuff

I just wanna press Q key and do this (&A_&Z_^A_^Z) (without any other key like Control or Shift, just raw)

**Comment by MAnDumu** [comment]
I figured out this: **COMBO Q [] > sequence (&A_&Z_^A_^Z)**

but it is not at the same time, it has a mini delay, and If I keep the key pressed it gets really laggy/delayed and keeps pressing A+Z for a few seconds
